packages/storage/src/database/general/database.ts

```diff
@@ -53,10 +58,13 @@ export class ModelDatabase<T extends Document> extends ModelBasic<T> {
     );
   }
 
-  public async createCollection(collectionName: string): Promise<void> {
+  public async createCollection(
+    collectionName: string,
+    session?: ClientSession
+  ): Promise<void> {
     const isExist = await this.isCollectionExist(collectionName);
```

Comment:
Can we pass session here?

Reply:
it just an checking code(get) that dont have any effect to the transaction block